common/src/types/v0/transport/cluster_agent.rs

```diff
@@ -83,6 +83,43 @@ impl ReportFailedPaths {
     }
 }
 
+/// Request struct to get Nvme subsystems registered for a given nqn.
+#[derive(Debug)]
+pub struct GetController {
+    nvme_path: String,
+}
+
+impl GetController {
+    /// Constructor to create instance of the struct. nvme_path is the device uri of the target
+    /// associated with volume.
+    pub fn new(nvme_path: String) -> Self {
```

Comment:
`associated` -> `Associated`

Reply:
I have added nvme path documentation in struct defination itself.